src/pages/management/configuration-api/index.mdx

```diff
@@ -2824,7 +2824,7 @@ Here's what you need to know about **webhooks**:
 
 Registers a webhook for the Client ID (application) provided in the request. One Client ID can register up to three webhooks for every `action`. The `license` webhooks'll be enabled automatically.
 
-As an alternative to calling Register Webhook, you can register webhooks via <Placeholder id="DEVELOPER_CONSOLE_URL" /> by configuring the Chat Webhooks building block.  [Learn more...](/getting-started/app-guides/chat-webhooks/)
+As an alternative to calling Register Webhook, you can register webhooks via Developer Console by configuring the Chat Webhooks building block.  [Learn more...](/getting-started/app-guides/chat-webhooks/)
```

Comment:
I have removed this link for now, as I and @AdalbertusLiver cannot figure out why it directs to the Dev Console email, not to the website. Looks like some tricky bug, so I will just remove it for now, as it was reported on #platform-docs